### PR TITLE
Login screen accessibility fixes

### PR DESCRIPTION
**Pull request by idevainstr**
refs: MBL-16969, MBL-16970
affects: Parent, Student, Teacher
release note: none.
test plan: See ticket.

## Screenshots

<table>
<tr><th>Before</th><th>After</th></tr>
<tr>
<td></td>
<td></td>
</tr>
<tr>
<td></td>
<td></td>
</tr>
</table>

## Checklist

- [ ] Follow-up e2e test ticket created
- [ ] A11y checked
- [ ] Tested on phone
- [ ] Tested on tablet
- [ ] Tested in dark mode
- [ ] Tested in light mode
- [ ] Approve from product

### CODE REVIEW

File: Core/Core/Assets.xcassets/Colors/electric.colorset/Contents.json
Comment (vargaat) [inline Core/Core/Assets.xcassets/Colors/electric.colorset/Contents.json:1]: These colors are generated from the `build-colors.js` script. I would advice against modifying colors there as they used across the applications and might break the contrast ratio on other places. Until we update the color system I suggest to apply a custom color for this screen's button.

**Review by vargaat — CHANGES_REQUESTED** [review]

**Comment by inst-danger** [comment]
<details><summary>Parent Build QR Code:</summary>

</details>

**Comment by inst-danger** [comment]
#### Release Note: 
none.

#### Affected Apps: Parent, Student, Teacher

[MBL-16969](https://instructure.atlassian.net/browse/MBL-16969)
[MBL-16970](https://instructure.atlassian.net/browse/MBL-16970)

Coverage | New % | Master % | Delta
-------- | ----- | -------- | -----
**Canvas iOS** | 92.41% | 92.42% | -0.01%
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 9ccdd8148b72647229a66d5a6b8b74814200c783
</p>

**Comment by inst-danger** [comment]
<details><summary>Student Build QR Code:</summary>

</details>

**Comment by inst-danger** [comment]
<details><summary>Teacher Build QR Code:</summary>

</details>

**Review by szabinst — APPROVED** [review]
QA + 1

**Review by vargaat — APPROVED** [review]
QA+1

**Pull request merged**